src/generators/crud/default/views/view.php

```diff
@@ -10,16 +10,17 @@
 
 /** @var \yii\db\ActiveRecord $model */
 /** @var $generator \schmunk42\giiant\generators\crud\Generator */
-
 ## TODO: move to generator (?); cleanup
 $model = new $generator->modelClass();
 $model->setScenario('crud');
 $safeAttributes = $model->safeAttributes();
-if (empty($safeAttributes)) {
+if (empty($safeAttributes))
+{
     $model->setScenario('default');
     $safeAttributes = $model->safeAttributes();
 }
-if (empty($safeAttributes)) {
+if (empty($safeAttributes))
+{
     $safeAttributes = $model->getTableSchema()->columnNames;
```

Comment:
`if` code should be PSR-2 compatible, see https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md